src/components/buttons/link-button/link-button.js

```diff
@@ -1,65 +1,110 @@
-        span {
-          color: ${props.isDisabled ? vars.colorNeutral : vars.colorPrimary};
-        }
-      `,
-      props.isDisabled &&
-        css`
-          cursor: not-allowed;
-        `,
-      !props.isDisabled &&
-        css`
-          &:hover {
-            span {
-              color: ${vars.colorPrimary25};
-            }
+const hoverStyles = () => css`
+  &:hover,
+  &:focus,
+  &:active {
+    span {
+      color: ${vars.colorPrimary25};
+    }
 
-            * {
-              fill: ${vars.colorPrimary25};
-            }
-          }
-        `,
-    ]}
-    onClick={props.isDisabled ? event => event.preventDefault() : undefined}
-    data-track-component="LinkButton"
-    {...filterAriaAttributes(props)}
-    {...filterDataAttributes(props)}
-    aria-label={props.label}
-  >
-    <Spacings.Inline scale="xs" alignItems="center">
-      {Boolean(props.iconLeft) &&
-        React.cloneElement(props.iconLeft, {
-          size: 'medium',
-          theme: props.isDisabled ? 'grey' : 'green',
-        })}
-      <Text.Body isInline={true}>{props.label}</Text.Body>
-    </Spacings.Inline>
-  </Link>
+    svg * {
+      fill: ${vars.colorPrimary25};
+    }
+  }
+`;
+
+const createStyledComponent = component => styled(component)`
+  display: inline-flex;
+  align-items: center;
+  font-size: 1rem;
+  border: none;
+  background: none;
+  padding: 0;
+  min-height: initial;
+  text-decoration: none;
+
+  span {
+    color: ${props => (props.disabled ? vars.colorNeutral : vars.colorPrimary)};
+  }
+
+  cursor: ${props => (props.disabled ? 'not-allowed' : 'pointer')};
+
+  ${props => !props.disabled && hoverStyles}
+`;
+
+const StyledReactRouterLink = createStyledComponent(ReactRouterLink);
+const StyledExternalLink = createStyledComponent('a');
+
+const LinkBody = props => (
+  <Spacings.Inline scale="xs" alignItems="center">
+    {Boolean(props.iconLeft) &&
+      React.cloneElement(props.iconLeft, {
+        size: 'medium',
+        theme: props.isDisabled ? 'grey' : 'green',
+      })}
+    <Text.Body isInline={true}>{props.label}</Text.Body>
+  </Spacings.Inline>
 );
 
+LinkBody.displayName = 'LinkBody';
+LinkBody.propTypes = {
+  iconLeft: PropTypes.element,
+  label: PropTypes.string,
+  isDisabled: PropTypes.bool,
+};
+
+const LinkButton = props => {
+  const remainingProps = getPassThroughProps(
+    props,
+    Object.keys(LinkButton.propTypes)
+  );
+
+  if (props.isExternal) {
+    return (
+      <StyledExternalLink
+        href={props.to}
+        onClick={props.isDisabled ? event => event.preventDefault() : undefined}
+        disabled={props.isDisabled}
+        data-track-component="LinkButton"
+        aria-label={props.label}
+        {...remainingProps}
+      >
+        <LinkBody
+          iconLeft={props.iconLeft}
+          isDisabled={props.isDisabled}
+          label={props.label}
+        />
+      </StyledExternalLink>
+    );
+  }
+
+  return (
+    <StyledReactRouterLink
+      to={props.to}
```

Comment:
This can be omitted from here because it's also passed in `remainingProps`, right?

Reply:
no, it's removed from remainingProps because it's a propType right?

Reply:
Ah yes, you're right!